test/e2e/storage/podlogs/podlogs.go

```diff
@@ -80,7 +80,21 @@ var expectedErrors = regexp.MustCompile(`container .* in pod .* is (terminated|w
 // log output of a container was already captured, then capturing does not
 // resume if the pod is marked for deletion.
 func CopyAllLogs(ctx context.Context, cs clientset.Interface, ns string, to LogOutput) error {
-	watcher, err := cs.CoreV1().Pods(ns).Watch(context.TODO(), meta.ListOptions{})
+	return CopyPodLogs(ctx, cs, ns, "", to)
+}
+
+// CopyPodLogs is basically the same as CopyAllLogs execpt that if podName is not empty, it only
```

Comment:
I'd move the comment from CopyAllLogs to here, changing what's necessary, and then the comment for CopyAllLogs can be just "CopyPodLogs for all current or future pods".

Reply:
updated. PTAL